modules/ai/assets/js/editor/components/prompt-history/history-modal.js

```diff
@@ -38,10 +39,12 @@ const PromptHistoryModal = ( props ) => {
 		deleteItem,
 	} = useDeletePromptHistoryItem();
 
+	const { quota } = useUserInfo();
+
 	const error = historyFetchingError || historyDeletingError;
 	const isLoading = isHistoryFetchingInProgress || isDeletingInProgress;
 	const isLastPage = meta && meta?.currentPage === meta?.totalPages;
-	const showUpgrade = items?.length > 0 && meta?.allowedDays < 90 && isLastPage;
+	const showUpgrade = quota < QUOTA_UPGRADE_THRESHOLD && items?.length > 0 && meta?.allowedDays < 90 && isLastPage;
```

Comment:
can we check the subscription type instead?

Reply:
we don't get it :/